src/platform/crypt_openssl.c

```diff
@@ -27,20 +27,102 @@
 #include "openssl/hmac.h"
 #endif
 #include "openssl/err.h"
+#include "openssl/evp.h"
 #include "openssl/kdf.h"
 #include "openssl/pem.h"
 #include "openssl/pkcs12.h"
 #include "openssl/pkcs7.h"
 #include "openssl/rsa.h"
 #include "openssl/ssl.h"
 #include "openssl/x509.h"
+
 #ifdef _WIN32
 #pragma warning(pop)
 #endif
 #ifdef QUIC_CLOG
 #include "crypt_openssl.c.clog.h"
 #endif
 
+EVP_CIPHER *CXPLAT_AES_128_GCM_ALG_HANDLE;
```

Comment:
Are these thread-safe?

Reply:
Yes, they are readonly after they are created which is at initialisation time.